docs/root/configuration/http/http_filters/local_rate_limit_filter.rst

```diff
@@ -0,0 +1,51 @@
+.. _config_http_filters_local_rate_limit:
+
+Local rate limit
+================
+
+* Local rate limiting :ref:`architecture overview <arch_overview_local_rate_limit>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.local_ratelimit.v3.LocalRateLimit>`
+* This filter should be configured with the name *envoy.filters.http.local_ratelimit*.
+
+The HTTP local rate limit filter applies a :ref:`token bucket
+<envoy_v3_api_field_extensions.filters.http.local_ratelimit.v3.LocalRateLimit.token_bucket>` rate
+limit when the request's route or virtual host has a per filter
+:ref:`local rate limit configuration <envoy_v3_api_msg_extensions.filters.http.local_ratelimit.v3.LocalRateLimit>`.
+
+If the local rate limit token bucket is checked, and there are no token availables, a 429 response is returned
+(the response is configurable). The local rate limit filter also sets the
+:ref:`x-envoy-ratelimited<config_http_filters_router_x-envoy-ratelimited>` header. Additional response
+headers may be configured.
+
+Statistics
+----------
+
+The local rate limit filter outputs statistics in the *cluster.<route target cluster>.local_ratelimit.* namespace.
```

Comment:
How does stat prefix come into play? It seems like we should be emitting on that tree vs. the cluster tree?

Reply:
Yeah, this wasn't up to date.